tools/src/bin/benchmark-clvm-cost.rs

```diff
@@ -535,6 +605,13 @@ pub fn main() {
             println!("   time: per-byte: {time_per_byte:.2}ns");
             println!("   cost: per-byte: {:.0}", time_per_byte * cost_scale);
             time_per_byte
+        } else if (op.flags & ALT_PER_BYTE_COST) != 0 {
+            let mut output = maybe_open(options.plot, op.name, "per-byte.log");
+            let (slope, _intercept) = time_per_byte_for_atom(&mut a, &mut output);
+            let cost = slope * cost_scale;
+            println!("   time: per-byte: {slope:.2}ns");
+            println!("   cost: per-byte: {:.0}", cost);
```

Comment:
```suggestion
            println!("   time: per-32bytes: {slope:.2}ns");
            println!("   cost: per-32bytes: {:.0}", cost);
```
Maybe something like this?